packages/core/test/Asset/util/isValidHandle.test.ts

```diff
@@ -8,10 +8,13 @@ const validHandles = [
   'test.handle',
   'test-handle-123',
   'test-handle-123.456',
-  'handle_name'
+  'handle_name',
+  'handle_name',
+  '@alice',
+  'test-@handle'
 ];
 
-const invalidHandles = ['bob!', '$wallet', 'alice@', 'test*handle#2', 'lace&bob', 'ada%1_test'];
+const invalidHandles = ['bob!', '$wallet', 'alice@', 'test*handle#2', 'lace&bob', 'ada%1_test', 'lace ', 'wallet '];
```

Comment:
how about this one? :thinking: 
```suggestion
const invalidHandles = ['bob!', '$wallet', 'alice@', 'test*handle#2', 'lace&bob', 'ada%1_test', 'lace ', 'wallet ', 'sub@sub@handle'];
```

Reply:
Wouldn't this be a sub-sub-handle ?

Reply:
Yes. I would think it's invalid too? I don't know, but it would be great to put this example to one of the arrays :sweat_smile:

Reply:
addressed: 8890ccc43abe351972e53dcb5d37ce31744d329f

Reply:
Addressed: 187d174dc745b1e12f223d88975d7e0da60ce538